chia/consensus/blockchain.py

```diff
@@ -98,6 +99,10 @@ class Blockchain(BlockchainInterface):
     compact_proof_lock: asyncio.Lock
     hint_store: HintStore
 
+    # Subscriptions for light clients
+    coin_subscriptions: Dict[bytes32, Set[bytes32]]  # Puzzle Hash : Set[Peer ID]
```

Comment:
This is not blockchain related, it's full node related, so it should be in the FullNodeStore